app/lib/katello/concerns/base_template_scope_extensions.rb

```diff
@@ -35,6 +40,22 @@ def host_products(host)
         host_subscription_facet(host)&.installed_products
       end
 
+      def host_products_names(host)
+        if products = host_products(host)
```

Comment:
can this ever return nil? I assume it's either empty array or array with records. given `[].join(', ') # => ""` this whole method can be a one liner

```
def host_products_names(host)
  products.map(&:name).join(', ')
end
```

unless I'm missing something

Reply:
It can be nil if the host doesn't have a subscription facet

Reply:
correct, ignore me then